polars/polars-sql/src/functions.rs

```
@@ -365,6 +373,21 @@ impl SqlFunctionVisitor<'_> {
             Log1p => self.visit_unary(Expr::log1p),
             Log2 => self.visit_unary(|e| e.log(2.0)),
             Pow => self.visit_binary::<Expr>(Expr::pow),
+            Round => match function.args.len() {
+                1 => self.visit_unary(|e| e.round(0)),
+                2 => self.visit_binary(|e, decimals| {
+                    e.round(match decimals {
+                        Expr::Literal(LiteralValue::Int64(n)) => n as u32,
+                        _ => {
+                            panic!("Invalid 'decimals' for Round: {}", function.args[1]);
```

I think we should return an error here. We can use the `polars_ensure` macro